packages/ag-charts-community/src/chart/legend/legend.test.ts

```diff
@@ -48,7 +48,7 @@ const OPTIONS: AgCartesianChartOptions = {
     series: SERIES,
 };
 
-function agChartsLogo({ path, size }: AgMarkerShapeFnParams) {
+function agChartsLogo({ path, size, x, y }: AgMarkerShapeFnParams) {
```

Comment:
I suspect this might be a breaking change. It’s definitely the right thing to do for our built-in markers. I’ll help you look into how we can avoid breaking customers markers

Reply:
Ignore this - this is correct